catroid/src/org/catrobat/catroid/content/bricks/ChangeVariableBrick.java

```diff
@@ -50,17 +49,16 @@
 import org.catrobat.catroid.ui.adapter.DataAdapter;
 import org.catrobat.catroid.ui.adapter.UserVariableAdapterWrapper;
 import org.catrobat.catroid.ui.dialogs.NewDataDialog;
-import org.catrobat.catroid.ui.dialogs.NewDataDialog.NewVariableDialogListener;
 import org.catrobat.catroid.ui.fragment.FormulaEditorFragment;
 
 import java.util.List;
 
-public class ChangeVariableBrick extends FormulaBrick implements NewVariableDialogListener {
+
+public class ChangeVariableBrick extends UserVariableBrick implements View.OnClickListener {
+
```

Comment:
It's not about a single newline. I'm fine with that. But there is no need for two single newlines in a row. See:
https://github.com/obusher/Catroid/blob/UVUL/catroid/src/org/catrobat/catroid/content/bricks/ChangeVariableBrick.java#L55-L59